spec/forms/support_interface/conditions_form_spec.rb

```diff
@@ -216,20 +216,26 @@
     end
 
     it 'reads standard and further conditions' do
-      application_choice = build(
-        :application_choice,
-        offer: { 'conditions' => ['Fitness to train to teach check', 'Get a haircut'] },
```

Comment:
There are a few other instances of `'conditions' => '...'` in this spec. It would be nice to ensure we are testing the new behaviour everywhere and not relying on the old offer implementation

Reply:
Removed